pootle/apps/pootle_app/management/commands/update_stores.py

```diff
@@ -45,9 +45,18 @@ def handle_translation_project(self, translation_project, **options):
 
     def handle_store(self, store, **options):
         overwrite = options.get('overwrite', False)
-        force = options.get('force', False)
-
-        store.update(overwrite=overwrite, only_newer=not force)
+        disk_mtime = store.get_file_mtime()
+        if disk_mtime is None:
+            logging.debug(u"File missing from disk, skipping "
+                          u"%s" % store.pootle_path)
+            return
+        if not options.get('force', False):
```

Comment:
>  And it should be options.force when you actually need to use the setting.
"force" is a dictionary key not an attribute so its not possible to do `options.force` only `options['force']`